### PR TITLE
[dash] Fix the issue with bulk creation of CA to PA and PA validation entries.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**
**What I did**
Fix the issue with the bulk creation of CA to PA and PA validation entries.

**Why I did it**
Before the fix, DashVnetOrch::addVnetMap always returned false after the creation of CA to PA and PA validation entries.
Which was considered by the calling DashVnetOrch::doTaskVnetMapTable method as an indication that the entry wasn't created. As a result, DashVnetOrch::doTaskVnetMapTable didn't remove the entry from the sync map (consumer.m_toSync) and tried to add the same entry again. Which caused an error and orchagent crash in an attempt to create the same CA to PA and PA validation entries in ASIC DB again.

**How I verified it**
Load the following configuration with `swssconfig` utility:
```
[
    {
      "DASH_APPLIANCE_TABLE:appliance-1": {
        "sip": "221.0.0.1",
        "vm_vni": "1"
      },
      "OP": "SET"
    },
    {
        "DASH_VNET_TABLE:vnet-1": {
          "vni": 1
        },
        "OP": "SET"
    },
    {
        "DASH_VNET_MAPPING_TABLE:vnet-1:1.4.0.1": {
            "routing_type": "vnet_encap",
            "underlay_ip": "221.2.0.1",
            "mac_address": "00:1B:6E:00:00:01",
            "use_dst_vni": "true"
        },
    "OP": "SET"
    },
    {
        "DASH_VNET_MAPPING_TABLE:vnet-1:1.4.0.2": {
            "routing_type": "vnet_encap",
            "underlay_ip": "221.2.0.1",
            "mac_address": "00:1B:6E:00:00:02",
            "use_dst_vni": "true"
        },
    "OP": "SET"
    },
    {
        "DASH_VNET_MAPPING_TABLE:vnet-1:1.4.0.3": {
            "routing_type": "vnet_encap",
            "underlay_ip": "221.2.0.1",
            "mac_address": "00:1B:6E:00:00:03",
            "use_dst_vni": "true"
        },
    "OP": "SET"
    },
    {
        "DASH_VNET_MAPPING_TABLE:vnet-1:1.4.0.4": {
            "routing_type": "vnet_encap",
            "underlay_ip": "221.2.0.1",
            "mac_address": "00:1B:6E:00:00:04",
            "use_dst_vni": "true"
        },
    "OP": "SET"
    }
]
```

**Details if related**

### CODE REVIEW

**Review by prsunny — APPROVED** [review]
lgtm. @prabhataravind to review/merge

File: orchagent/dash/dashvnetorch.cpp [hunk 1]
```diff
@@ -333,12 +333,13 @@ bool DashVnetOrch::addVnetMap(const string& key, VnetMapBulkContext& ctxt)
         {
             addOutboundCaToPa(key, ctxt);
             addPaValidation(key, ctxt);
+            return true;
```
Comment (prabhataravind) [inline orchagent/dash/dashvnetorch.cpp:336]: I think we want to remove the entries from consumer.m_toSync only after flushing the bulk entries. This is handled later when addVnetMapPost returns true. 

if (addVnetMapPost(key, ctxt))
                {
                    it_prev = consumer.m_toSync.erase(it_prev);
                }
                else
                {
                    it_prev++;
                }

**Review by prabhataravind — COMMENTED** [review]
Let me reproduce this to understand the issue better

File: orchagent/dash/dashvnetorch.cpp (hunk 1 above)
Comment (oleksandrivantsiv) [reply orchagent/dash/dashvnetorch.cpp:336]: The PR is not fully resolving the issue. The changes contradict the logic shown above. 
@prabhataravind I'm closing this PR and will open an issue instead.

**Pull request closed without merging**